nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/documentation/feature/_documentation.component-theme.scss

```
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'sass:map';
+@use '@angular/material' as mat;
+
+@mixin generate-theme($material-theme, $config) {
+    $is-material-dark: if(mat.get-theme-type($material-theme) == dark, true, false);
+
+    $material-theme-tertiary-palette-darker: mat.get-theme-color(
+        $material-theme,
+        tertiary,
+        map.get(map.get($config, tertiary), darker)
+    );
+    $material-theme-tertiary-palette-lighter: mat.get-theme-color(
+        $material-theme,
+        tertiary,
+        map.get(map.get($config, tertiary), lighter)
+    );
+
+    $selected-item-color: if(
+        $is-material-dark,
+        $material-theme-tertiary-palette-darker,
+        $material-theme-tertiary-palette-lighter
+    );
+
+    .documentation {
```

IMO the `h2` in the documentation should use some additional bottom margin:

```suggestion
    .documentation {
         h2 {
            margin-bottom: 12px !important;
        }
```